lib/openxml/docx/elements/word_processing_shapes_shape_properties.rb

```diff
@@ -0,0 +1,23 @@
+module OpenXml
+  module Docx
+    module Elements
+      class WordProcessingShapesShapeProperties < Container
+        tag :spPr
+        namespace :wps
+
+        attribute :bwMode, expects: :valid_colors
+
+      private
+
+        def valid_colors(value)
+          raise ArgumentError unless ok_values.include? value
+        end
+
+        def ok_values
+          %i(auto black blackGray blackWhite clr gray grayWhite hidden invGray ltGray white)
+        end
```

Comment:
I added the [one_of] (https://github.com/openxml/openxml-docx/commit/64752383cb5e46f633160cb3b1ba3445ec581142#diff-04f2e3073e4c5bfcaac3120d9e04d4a0R206) argument to attribute builder and updated all the files that were using that type of validation.